src/open_inwoner/templates/pages/ssd/reports_base.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'master.html' %}
+{% load i18n grid_tags dropdown_tags form_tags button_tags ssd_tags render_tags %}
+
+{% block content %}
+    {% render_grid %}
+        {% render_column span=9 %}
+            <h1 class="utrecht-heading-1" id="title">
+                {% trans "Mijn uitkeringen" %}
+            </h1>
+            <div class="wysiwyg">
```

Comment:
What is this template for? Is this for the PDF? 
(if so: in that case we cannot use the wysiwyg class here; we will need the styling from the PDF CSS file).